### PR TITLE
New Term Entry: PHP String Functions `explode()`

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Description

Added term entry for PHP String Function `explode()` from Target Keyword list. Uses directory structure from [660](https://github.com/Codecademy/docs/pull/660)

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/string-functions/terms/explode/explode.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+- A `$limit` of zero is treated as 1.
+- A negative `$limit` will omit the last -`$limit` elements from the result.
+- If `$limit` is omitted, `explode()` will return as many substrings as possible.
+
+The `$delimiter` has the following behaviors:
+
+- If `$delimiter` is an empty string, `explode()` with throw a `ValueError` exception.
+- If `$delimiter` is not found in `$string` and a negative `$length` is used, and empty array is returned.
+- If `$delimiter` is not found in `$string` and `$length` is positive or omitted, an array containing `$string` is returned.
+- If `$delimiter` is found at the start or end of `$string` empty values for each `$delimiter` will be added to the start or end of the returned array respectively.
```
Comment (KTom101) [inline content/php/concepts/string-functions/terms/explode/explode.md:40]:
```suggestion
- If `$delimiter` is found at the start or end of `$string`, empty values for each `$delimiter` will be added to the start or end of the returned array respectively.
```

File: content/php/concepts/string-functions/terms/explode/explode.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The `explode()` function takes a string and splits it on a given delimiter string, and returns an array of the substrings produced.
+
+## Syntax
+
+```php
```
Comment (KTom101) [inline content/php/concepts/string-functions/terms/explode/explode.md:21]:
```suggestion
```pseudo
```

**Review by KTom101 — CHANGES_REQUESTED** [review]
@SSwiniarski I've made a review of this.

**Comment by SSwiniarski** [comment]
@KTom101, I implemented your suggestions.

**Review by KTom101 — APPROVED** [review]
@SSwiniarski Looks good for the next review.

File: content/php/concepts/string-functions/terms/explode/explode.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  $strarr3 = explode(",",$str,-2);
+
+  var_dump($strarr1);
+  echo("<br/>");
+  var_dump($strarr2);
+  echo("<br/>");
+  var_dump($strarr3);
+  echo("<br/>");
+?>
+
```
Comment (Dusch4593) [inline content/php/concepts/string-functions/terms/explode/explode.md:58]:
```suggestion
?>\
```

File: content/php/concepts/string-functions/terms/explode/explode.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+The `$delimiter` has the following behaviors:
+
+- If `$delimiter` is an empty string, `explode()` will throw a `ValueError` exception.
+- If `$delimiter` is not found in `$string` and a negative `$length` is used, an empty array is returned.
+- If `$delimiter` is not found in `$string` and `$length` is positive or omitted, an array containing `$string` is returned.
+- If `$delimiter` is found at the start or end of `$string`, empty values for each `$delimiter` will be added to the start or end of the returned array respectively.
+
+## Codebyte Example
+
```
Comment (Dusch4593) [inline content/php/concepts/string-functions/terms/explode/explode.md:43]: Let's a brief sentence here that describes what's happening in the codebyte below:

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@SSwiniarski Finished my review and left you a couple comments/suggestions.

**Comment by SSwiniarski** [comment]
@Dusch4593 I've implemented your suggestion.

File: content/php/concepts/string-functions/terms/explode/explode.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+```
+
+The `$delimiter` parameter specifies the string on which `$string` will be split.
+The `$string` parameter is the string that will be split by the `explode()` function.
+
+The `$limit` parameter is an optional `int` that specifies how many strings will be returned in the resulting array as follows:
+
+- A positive value will return up to `$limit` strings, if more than `$limit` strings can be produced splitting on `$delimiter`, the last element will contain the remainder of `$string`.
+- A `$limit` of zero is treated as 1.
+- A negative `$limit` will omit the last -`$limit` elements from the result.
```
Comment (caupolicandiaz) [inline content/php/concepts/string-functions/terms/explode/explode.md:32]:
```suggestion
- A negative `$limit` will omit the last n elements from the result.
```

Comment (caupolicandiaz) [reply content/php/concepts/string-functions/terms/explode/explode.md:32]: I found this description confusing, perhaps revise to something similar to the suggestion above

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@SSwiniarski looks pretty good, just one comment

**Comment by SSwiniarski** [comment]
@caupolicandiaz, I implemented your suggestion.

**Review by Dusch4593 — APPROVED** [review]
LGTM! 👍🏻

**Comment by SSwiniarski** [comment]
@caupolicandiaz, do we look ok to merge now?

File: content/php/concepts/string-functions/terms/explode/explode.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+Tags:
+  - 'Arrays'
+  - 'Strings'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The `explode()` function takes a string and splits it on a given delimiter string, and returns an array of the substrings produced.
```
Comment (yangc95) [inline content/php/concepts/string-functions/terms/explode/explode.md:17]:
```suggestion
The `explode()` function takes a string and splits it on a given delimiter string. It returns an array of the substrings produced.
```

**Comment by SSwiniarski** [comment]
@Dusch4593, I took care of it.

**Comment by Dusch4593** [comment]
> @caupolicandiaz, do we look ok to merge now?

Re-bump.

**Review by yangc95 — APPROVED** [review]

**Review by caupolicandiaz — APPROVED** [review]

**Comment by Dusch4593** [comment]
@SSwiniarski This looks good for merge 😄 I made a small edit in 41b3fad24707bb029f3606818143f10ea2aec4f1 where I removed an empty line and added spacing in the Codebyte snippet.

**Comment by Dusch4593** [comment]
Live entry here: https://www.codecademy.com/resources/docs/php/string-functions/explode

**Pull request merged**